tcms/testplans/tests/test_new_plan.py

```diff
@@ -76,36 +118,117 @@ def test_plan_create_new_inactive(self):
         self._test_plan_create_new(is_active=False)
 
     def _test_plan_create_new(self, is_active):
-        self.request['is_active'] = is_active
+        self.request["is_active"] = is_active
```

Comment:
why do we need this change ?

Reply:
I don't quite understand, do you mean why do we need the test or why the double quotes instead of single quotes?